packages/cli/commands/project/dev.js

```diff
@@ -119,6 +129,37 @@ exports.handler = async options => {
     targetProjectAccountId = hasPublicApps ? parentAccountId : targetAccountId;
     targetTestingAccountId = targetAccountId;
 
+    // Only used for developer test accounts that are not yet in the config
+    if (notInConfigAccount) {
+      const useExistingDevTestAcct = await confirmUseExistingDeveloperTestAccountPrompt(
+        notInConfigAccount
+      );
+      if (!useExistingDevTestAcct) {
+        logger.log('');
+        logger.log(
+          i18n(
+            `cli.lib.localDev.confirmDefaultAccountIsTarget.declineDefaultAccountExplanation`,
+            {
+              useCommand: uiCommandReference('hs accounts use'),
+              devCommand: uiCommandReference('hs project dev'),
+            }
+          )
+        );
+        logger.log('');
+        process.exit(EXIT_CODES.SUCCESS);
+      }
+      const devTestAcctConfigName = await saveDevTestAccountToConfig(
+        env,
+        notInConfigAccount
+      );
+      logger.success(
+        i18n(`cli.lib.developerTestAccount.create.success.configFileUpdated`, {
+          accountName: devTestAcctConfigName,
+          authType: PERSONAL_ACCESS_KEY_AUTH_METHOD.name,
+        })
+      );
```

Comment:
Trying to keep things as concise as possible within this file - could you move this logic to `lib/localDev`?

Reply:
yep was just thinking about this, will push that up in a bit